spec/system/flatpickr_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "system_helper"
+
+describe "Test Flatpickr", js: true do
+  include AuthenticationHelper
+  include WebHelper
+
+  context "orders" do
+    it "opens the datepicker and closes it using the last button (the 'Close' one)" do
+      login_as_admin_and_visit 'admin/orders'
+      open_datepicker('#q_completed_at_gteq')       
+      # Looks for the close button and click it
+      within(".flatpickr-calendar.open") do
+        expect(page).to have_selector '.shortcut-buttons-flatpickr-buttons'
+        find(".shortcut-buttons-flatpickr-buttons > button:nth-last-child(1)").click
+      end
+      # Should no more have opened flatpickr
+      expect(page).not_to have_selector '.flatpickr-calendar.open'
+    end
+    it "opens the datepicker and sets date to today" do
```

Comment:
We usually have empty lines in between examples.
```suggestion
    end

    it "opens the datepicker and sets date to today" do
```